cffi/build.py

```diff
@@ -30,7 +30,7 @@ def search_paths(env_var: str) -> List[str]:
 EXTRA_CFLAGS += shlex.split(os.environ.get("LIBYANG_EXTRA_CFLAGS", ""))
 EXTRA_LDFLAGS = shlex.split(os.environ.get("LIBYANG_EXTRA_LDFLAGS", ""))
 
-with open(os.path.join(HERE, "source.c")) as f:
+with open(os.path.join(HERE, "source.c"), encoding="utf-8") as f:
```

Comment:
Did you have a problem with the default encoding? These files do not contain any non-ASCII characters as far as I know.

Reply:
I added it to make pylint happy (it is a new warning added recently):

- unspecified-encoding (W1514) (https://pylint.pycqa.org/en/latest/technical_reference/features.html)
- https://pylint.pycqa.org/en/latest/whatsnew/2.10.html

Alternatively, I can disable that specific warning, either inline or altogether in pylintrc. What are your thoughts on this?

Reply:
Ok I didn't get that was the pylint warning. This is good, leave it :+1: